state/disabled/disabledTrackableDataTrie_test.go

```diff
@@ -0,0 +1,65 @@
+package disabled
+
+import (
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewDisabledTrackableDataTrie(t *testing.T) {
+	t.Parallel()
+
+	t.Run("new disabledTrackableDataTrie", func(t *testing.T) {
+		t.Parallel()
+
+		assert.False(t, check.IfNil(NewDisabledTrackableDataTrie()))
+	})
+
+	t.Run("retrieve value", func(t *testing.T) {
```

Comment:
might extract these in other functions, since the big test name (`TestNewDisabledTrackableDataTrie`) only indicated the testing of the constructor

Reply:
Done.